UPGRADE-3.0.md

```diff
@@ -233,6 +233,8 @@ UPGRADE FROM 2.x to 3.0
 
 ### Form
 
+ * The `max_length` option was removed. Use the `attr` option instead by setting it to an `array` with a `maxlength` key.
```

Comment:
Could you add a line break to make this line shorter and match the others? A line break after `it to` looks about right